lookout/style/format/analyzer.py

```diff
@@ -172,7 +172,16 @@ def _files_by_language(cls, files: Iterable[File]) -> Dict[str, Dict[str, File]]
         return result
 
     @classmethod
-    def _load_config(cls, config: Mapping[str, Any]):
+    def _load_analyze_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
```

Comment:
please use the same mechanism as in train_config. Probably it's best to make recursive_update an external function to avoid duplicates.

Reply:
yeah, it is not necessary right now so, I wanted to avoid it, but ok.

Reply:
I agree but it's not long to do and it's good to have consistency.